cluster/gce/gci/configure-kubeapiserver.sh

```diff
@@ -113,6 +113,9 @@ function start-kube-apiserver {
   if [[ -n "${SERVICEACCOUNT_CERT_PATH:-}" ]]; then
     params+=" --service-account-key-file=${SERVICEACCOUNT_CERT_PATH}"
   fi
+  if [[ -f "/etc/srv/kubernetes/known_tokens.csv" ]]; then
+    chown "${KUBE_API_SERVER_RUNASUSER:-0}":"${KUBE_API_SERVER_RUNASGROUP:-0}" "/etc/srv/kubernetes/known_tokens.csv"
```

Comment:
and
```shell
chown "${KUBE_API_SERVER_RUNASUSER:-0}":"${KUBE_API_SERVER_RUNASGROUP:-0}" '/etc/srv/kubernetes/known_tokens.csv'
```

Or perhaps have a variable that contains `/etc/srv/kubernetes/known_tokens.csv` instead since it is used more than once?

Reply:
Updated to use a variable as your suggested.